.gitignore

```diff
@@ -1,2 +1,3 @@
 .idea
-**/.DS_Store
\ No newline at end of file
+**/.DS_Store
+node_modules
```

Comment:
not needed we dont have node modules in the root directory. you probably accidentally executed yarn in the root. you can delete the packes.json files in the root directory

Reply:
genau das isch passiert, wenn ich mich richtig erinnere sogar de martin :).

Reply:
Ja im au er het node modules im root pushed gha 😅. Ich hans aber grad vorher schnell glöscht. jetzt muesch es nur no bi dir au lösche.